stable/node-problem-detector/README.md

```diff
@@ -34,21 +34,22 @@ Custom System log monitor config files can be created, see [here](https://github
-| `nameOverride`          | Override the name of the chart             | `nil`                                                        |
-| `rbac.create`           | RBAC                                       | `true`                                                       |
-| `resources`             | Pod resource requests and limits           | `{}`                                                         |
-| `settings.log_monitors` | System log monitor config files            | `/config/kernel-monitor.json`, `/config/docker-monitor.json` |
-| `serviceAccount.create` | Whether a ServiceAccount should be created | `true`                                                       |
-| `serviceAccount.name`   | Name of the ServiceAccount to create       | Generated value from template                                |
-| `tolerations`           | Optional daemonset tolerations             | `[]`                                                         |
+| Parameter                         | Description                                | Default                                                      |
+| ----------------------------------|--------------------------------------------|--------------------------------------------------------------|
+| `affinity`                        | Map of node/pod affinities                 | `{}`                                                         |
+| `annotations`                     | Optional daemonset annotations             | `{}`                                                         |
+| `fullnameOverride`                | Override the fullname of the chart         | `nil`                                                        |
+| `image.pullPolicy`                | Image pull policy                          | `IfNotPresent`                                               |
+| `image.repository`                | Image                                      | `k8s.gcr.io/node-problem-detector`                           |
+| `image.tag`                       | Image tag                                  | `v0.6.1`                                                     |
+| `nameOverride`                    | Override the name of the chart             | `nil`                                                        |
+| `rbac.create`                     | RBAC                                       | `true`                                                       |
+| `resources`                       | Pod resource requests and limits           | `{}`                                                         |
+| `settings.log_monitors`           | System log monitor config files            | `/config/kernel-monitor.json`, `/config/docker-monitor.json` |
+| `settings.custom_plugin_monitors` | Custom plugin monitor config files         | `/config/kernel-monitor.json`, `/config/docker-monitor.json` |
```

Comment:
I think you've copy and pasted this from above but not changed the default value. It should be `[]`